server/src/AMQPConnectionInfo.cc

```diff
@@ -32,6 +34,7 @@ using namespace mlpl;
 struct AMQPConnectionInfo::Impl {
 	Impl()
 	: m_URL(),
+	  m_URLBuf(NULL),
```

Comment:
m_URLBuf is only used in setURL(), isn't it.
If so, it's unnecessary to be an instance variable.
Or will you use it in the methods you are going to make later.